install_config/install/advanced_install.adoc

```diff
@@ -1311,12 +1311,20 @@ not set, then cluster logging data is stored in an `emptyDir` volume, which will
 be deleted when the Elasticsearch pod terminates.
 
 There are three options for enabling cluster logging storage when using the
-advanced install:
+advanced installation; however, for production environments, you should use only block storage.
+Do not attempt to use NFS storage. NFS storage is described below only to indicate how it might be 
+used if you were not using block storage, or you were not in a production environment.
+
 
 [discrete]
 [[advanced-installation-logging-storage-nfs-host-group]]
 ===== Option A: NFS Host Group
 
+[NOTE]
+====
+We recommend that you do not attempt to use NFS storage in a production environment.
```

Comment:
I know that "we recommend" vs. "it is recommended" vs. "Red Hat recommends" vs. avoiding recommend at all is a big discussion topic and a little hazy guidance, but I definitely think we should avoid "we recommend", since we really never use "we" in the docs.

I'd suggest to either say "It is recommended", since that is already used a few places in this file, or just say "Do not", like how you worded it in the above paragraph.

Same for the admonition below.

Reply:
Changed to "Do not use NFS...." Thanks.

Reply:
Changed in both places. Thanks.